metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/MaaSFunctions.java

```diff
@@ -188,15 +189,27 @@ public void initialize(Context context) {
       catch(Exception ex) {
         LOG.error(ex.getMessage(), ex);
       }
+      finally {
+        isInitialized = true;
```

Comment:
That makes sense.  Maybe a little comment to that effect, will prevent me from asking you this same question in a few months.  Ha.